lib/pharos/kubeconfig_command.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'pharos/up_command'
+require 'pharos/ssh/client'
+require 'pharos/ssh/manager'
+require 'pharos/yaml_file'
+require 'pharos/terraform/json_parser'
+require 'pharos/kube/config'
+
+module Pharos
+  class KubeconfigCommand < UpCommand
+    option ['-n', '--name'], 'NAME', 'Overwrite cluster name', attribute_name: :new_name
+    option ['-C', '--context'], 'CONTEXT', 'Overwrite context name', attribute_name: :new_context
+    option ['-m', '--merge'], '[FILE]', 'Merge with existing configuration file', multivalued: true
+
+    REMOTE_FILE = "/etc/kubernetes/admin.conf"
+
+    def execute
+      config = Pharos::Kube::Config.new(config_file_content)
+      config.rename_cluster(new_name) if new_name
+      config.rename_context(new_context) if new_context
+      config.update_server_address(master_host.api_address)
+      merge_list.each do |merge|
+        merge_config = Pharos::Kube::Config.new(File.read(merge))
```

Comment:
IMO it would make sense to also somehow support `KUBECONFIG` env, or the default location `~/.kube/config`.

Reply:
I mean support those in the merging